Survey Checks/README.md

```diff
@@ -0,0 +1,32 @@
+# Survey checks
```

Comment:
Should we call this iesurveychecks? or iesurveyhfcs? i think it's nice to keep with consistent branding of ie... and if they have a unique name will be easier to refer to each toolkit in other parts of the repo

Reply:
Other broad comments on the set-up / folder structure:

1. Would be clearer to have a list of the type of checks performed on the main readme, so it's clearer from the outset what the dashboard is designed to do
2.  i think it would be better to separate the data quality checks from the survey completeness checks at a higher level. So maybe this folder is called survey completeness checks, and another folder (at the same level of the hierarchy) is called data quality checks. there are pros and cons to splitting, but right now the structure is confusing. If keeping them in the same folder, i think at least we should have subfolders for each and have the main readme explaining the overall flow of how completeness checks and data quality checks fit together, and then detailed instruction about how to implement each in their own repos. the main downside i see to doing this is that the webinar will hopefully pull lots of people to the repo next week and we will definitely not have the data quality checks up and running, so i think better if that is a separate folder that we can clearly specify as a work-in-progress.